mods/persistence/modules/overmap/_overmap.dm

```diff
@@ -1,30 +1,30 @@
-// Complete override since Persistence hotloads it maps, and most of these functions must be delayed.
```

Comment:
If this has been depreciated you can delete the file

Reply:
kk